extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -203,8 +231,35 @@ export function getConnectionProfile(
 	};
 }
 
-export async function collectSourceDatabaseTableInfo(sourceConnectionId: string, sourceDatabase: string): Promise<TableInfo[]> {
-	const ownerUri = await azdata.connection.getUriForConnection(sourceConnectionId);
+export async function getSourceConnectionString(): Promise<string> {
+	return await azdata.connection.getConnectionString((await getSourceConnectionProfile()).connectionId, true);
+}
+
+export async function getTargetConnectionString(
```

Comment:
Should we make this function more generic? Right now it is hard-coded for login migration settings. There is a risk that someone might use this in the future based on the function name without realize the effects on encrypt/trustservercertificate

Reply:
good call, will take the two bools as parameters